src/lyk.c

```diff
@@ -0,0 +1,94 @@
+#include "lyk.h"
+#include "lobject.h"
+#include "lopcodes.h"
+#include "ldebug.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include "assert.h"
+
+#ifdef USE_YK
+#include <yk.h>
+#endif
+
+/*
+ * Is the instruction `i` the start of a loop?
+ *
+ * YKFIXME: Numeric and Generic loops can be identified by OP_FORLOOP and OP_TFORLOOP opcodes. 
+ * Other loops like while and repeat-until are harder to identify since they are based on OP_JMP instruction.
+ */
+int is_loop_start(Instruction i)
+{
```

Comment:
Can we make the bracing style like the rest of the Lua code? Looks like they put the open brace on the same line as the parent.

Reply:
ok

Reply:
done :point_right: 9d00f0c7c8e12a34d6c3c7df5cfb135687790e61

Reply:
oops, I wasn't pushing to upstream. 
the right commit is here :point_right: 945f19d55b943440f56fec0e64a9aff8a741e2b8